File: 01.JS-Basic/ИЗПИТ/Deer of Santa.js
function deerOfSanta(input) {

    let daysAway = Number(input[0])
    let foodLefted = Number(input[1])
    let firstDeerFood = Number(input[2])
    let secondDeerFood = Number(input[3])
    let thirdDeerFood = Number(input[4])

    let totalfoodNeded = firstDeerFood * daysAway + secondDeerFood * daysAway + thirdDeerFood * daysAway

    if (totalfoodNeded <= foodLefted) {
        console.log(`${Math.ceil(foodLefted) - Math.ceil(totalfoodNeded)} kilos of food left.`)
    } else {
        console.log(`${Math.ceil(totalfoodNeded) - Math.ceil(foodLefted)} more kilos of food are needed.`)
    }
}
deerOfSanta(['5', '10', '2.1', '0.8', '11'])

File: JAVA-Basic/JavaBasic/src/IZPIT/DeerOfSanta.java
package IZPIT;

import java.util.Scanner;

public class DeerOfSanta {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int daysAway = Integer.parseInt(scan.nextLine());
        int foodLefted = Integer.parseInt(scan.nextLine());
        double firstDeerFood = Double.parseDouble(scan.nextLine());
        double secondDeerFood = Double.parseDouble(scan.nextLine());
        double thirdDeerFood = Double.parseDouble(scan.nextLine());

        double totalFoodNeeded = firstDeerFood * daysAway + secondDeerFood * daysAway + thirdDeerFood * daysAway;
        double kgDifference = Math.abs(totalFoodNeeded - foodLefted);

        if (totalFoodNeeded <= foodLefted) {
            System.out.printf("%.0f kilos of food left.", Math.floor(kgDifference));
        } else {
            System.out.printf("%.0f more kilos of food are needed.", Math.ceil(kgDifference));
        }
    }
}
